sme_finance_application_schema/finance_need_v1

```diff
@@ -8,6 +8,10 @@
         "deposit": {
             "type": "integer"
         },
+        "expectations": {
```

Comment:
Let's go broader here. Free text about the finance requirement, not just "what they expect or need"?